pom.xml

```diff
@@ -29,9 +29,65 @@
 		<jacoco.out.paths>${project.build.directory}</jacoco.out.paths>
 		<sonar.jacoco.reportPath>${jacoco.out.path}/jacoco.exec</sonar.jacoco.reportPath>
 		<jacoco.version>0.8.2</jacoco.version>
+		<log4j2.version>2.17.2</log4j2.version>
+		<spring.version>5.3.18</spring.version>
```

Comment:
spring version is `5.3.18`, not `5.2.20.RELEASE`?

Reply:
The minimum minor version update for this spring-boot release is 5.3.18.